static/index.html

```diff
@@ -88,9 +88,9 @@
     <div class="row">
         <div class="col-md-6">
             <div class="intro">
-                <p>Getting involved in politics doesn’t have to be complicated or confusing. When you join <em>Calls for Change</em>, we'll automatically pair you with your members of Congress and send you a text or email reminder to call them once a week. We’ll also send you information on a suggested topic that we think is important to women and equality more broadly.</p>
+                <p>It's easier than you think to do something to advance women's rights and quality broadly (<a herf="#about">more details</a>).  Calling members of Congress is the most effective way to influence the political process, and Calls for Change makes it <a href="how-it-works">simple and fast</a>.  Each week we'll send you all the infromation you need — from the right phone number to a sample script — so in just ten minutes you can make sure your voice is heard.</p>
```

Comment:
Do we have have a `how-it-works` page or is this supposed to be `#how-it-works`?